tools/cdb2sql/cdb2sql.c

```diff
@@ -887,6 +891,26 @@ static void replace_args(int argc, char *argv[])
     }
 }
 
+
+static int int_handler_cnt = 0;
+
+/* If ctrl_c was pressed to clear existing line and go to new line
```

Comment:
CTRL+C should make the control jump to the next (empty) prompt, after printing C^.  2 CTRL+C's should behave in a similar way, just jump an additional prompt.
CTRL-D, OTOH should kill cdb2sql iff the input buffer is empty, a NOOP otherwise.